files/default/slack_handler_webhook.rb

```diff
@@ -80,7 +80,12 @@ def slack_message(message, webhook, text_attachment = nil)
     http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
what do you think of disabling VERIFY_NONE? i.e always verify peers
or if you really need such behaviour, make it as attribute?